collections/concepts/what-are-tokens.md

```diff
@@ -12,64 +12,592 @@ image:
+
+```json
+{
+  "type": "token",
+  "data": "123-45-6789",
+  "deduplicate_token": true
+}
+```
+
+In this scenario, if we detect an existing token with the same `fingerprint`, the existing token is returned instead of creating a new token.
+
+## Metadata
+
+Being able to tag your tokens with custom attributes is important in many scenarios. For instance, you may want to add a system identifier to your tokens that allows you to reference a record in your own database. Another scenario is you need to tag records that fall into certain compliance requirements (e.g. GDPR). In these scenarios, putting this information in the token data may not be ideal as you want to be able to quickly reference the information or expose it to audiences who do not have access to view the token data. To solve for this, Basis Theory tokens allow you to set your own metadata on any token utilizing key-value pairs.
+
+**Request**
+
+```json
+{
+  "type": "token",
+  "data": "John Doe",
+  "metadata": {
+    "customer_id": "123abc"
+  }
+}
+```
+
+**Response**
+
+```json
+{
+  "id": "a2f1defa-da99-44e7-b70b-4e6dcfa20ec2",
+  "type": "token",
+  "metadata": {
+    "customer_id": "123abc"
+  }
+}
+```
+
+## Associations
+
+Associations enable you to build relationships between your data. You can construct your relationships into trees or chains enabling you to discover and traverse your data efficiently.
+
+You can easily manage the relationship between a `parent` and `child` token via our [token association API endpoints](https://docs.basistheory.com/#token-associations).
+
+# Use Cases
+
+## Tokenize Credit Cards
+
+**Request**
+
+{% raw %}
+```json
+{
+  "id": "{{ data.number | alias_preserve_format }}",
+  "type": "card",
+  "data": {
+    "number": "4242424242424242",
+    "expiration_month": 12,
+    "expiration_year": 2025,
+    "cvc": "123"
+  },
+  "fingerprint_expression": "{{ data.number }}",
+  "mask": {
+    "number": "{{ data.number | reveal_last: 4 }}",
+    "expiration_month": "{{ data.expiration_month }}",
+    "expiration_year": "{{ data.expiration_year }}"
+  },
+  "search_indexes": [
+    "{{ data.number }}",
+    "{{ data.number | last4 }}"
+  ],
+  "privacy": {
+    "classification": "pci",
+    "impact_level": "high",
+    "restriction_policy": "mask"
+  }
+}
+```
+{% endraw %}
+
+**Response**
+
+{% raw %}
+```json
+{
+  "id": "3829018309324938",
+  "type": "card",
+  "data": {
+    "number": "XXXX XXXX XXXX 4242",
+    "expiration_month": 12,
+    "expiration_year": 2025
+  },
+  "fingerprint": "M25bjMqlH85LZE2F7SmZ1w",
+  "fingerprint_expression": "{{ data.number }}",
+  "mask": {
+    "number": "{{ data.number | reveal_last: 4 }}",
+    "expiration_month": "{{ data.expiration_month }}",
+    "expiration_year": "{{ data.expiration_year }}"
+  },
+  "expires_at": {
+    "cvc": "2022-06-30T15:00:00+00:00"
```

Comment:
do we want to keep this as a part of the response? if so, I'll have to save that data when you first create and update that each time the `cvc` changes